From 1f6e710a8cdc3c4fb95af807330e05dfd5e7ca6c Mon Sep 17 00:00:00 2001
Message-Id: <1f6e710a8cdc3c4fb95af807330e05dfd5e7ca6c.1379425497.git.minovotn@redhat.com>
In-Reply-To: <f5f5558c6067d2cdb77e46707b0bda1f4f885402.1379425497.git.minovotn@redhat.com>
References: <f5f5558c6067d2cdb77e46707b0bda1f4f885402.1379425497.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 11 Sep 2013 18:43:01 +0200
Subject: [PATCH 25/25] qemu-img: fix invalid JSON

Single quotes for JSON are a QMP-ism, use real JSON in
qemu-img output.

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(not yet upstream)
Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-img.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qemu-img.c b/qemu-img.c
index 2842bbd..cf1ba0f 100644
--- a/qemu-img.c
+++ b/qemu-img.c
@@ -1340,7 +1340,7 @@ static void dump_map_entry(OutputFormat output_format, MapEntry *e,
                (e->flags & BDRV_BLOCK_ZERO) ? "true" : "false",
                (e->flags & BDRV_BLOCK_DATA) ? "true" : "false");
         if (e->flags & BDRV_BLOCK_OFFSET_VALID) {
-            printf(", 'offset': %"PRId64"", e->offset);
+            printf(", \"offset\": %"PRId64"", e->offset);
         }
         putchar('}');
 
-- 
1.7.11.7

